NSS-Day-Cohort-66/rare-client-golden-droids-1
# Zd rare client start

I have the initial fetch call, classes, and JSX for the My Posts page


**steps to test**
1. checkout to main and fetch branch
2. switch to zd-rare-client-start 
3. be sure that zd-rare-first-ticket branch is running on server side
4. open nav bar and click My Posts - should go to `http://localhost:3000/posts`
5. verify you see a list of posts in chronological order, with most recent first 
6. network tab should show 200 status
